### DISCUSSION TITLE
Heartbeat installation on Security Onion

### DISCUSSION THREAD

**Discussion by john-mathew-itt** [Security-Onion-Solutions/securityonion#6959] [Unsupported Versions] [answered]
Hi,
I am trying to install and configure heartbeat on Security Onion Forward nodes so that I can monitor reachability and Uptime of couple of my servers within my network. The idea is to view Uptime data in Security Onion Kibana for all monitored servers. 
![image](https://user-images.githubusercontent.com/62584618/150763410-7ce1456b-d20d-4b3a-875c-7d5d7669f41e.png)

Any suggestions on how I can achieve this.

**Security Onion version:** 2.3.90
**Is this a cloud deployment or on-prem?**
On-prem deployment
**Did you install from our Security Onion ISO image or did you perform a network installation?**
Official ISO image
**How many nodes do you have? How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)**
1 ManagerSearch node and 2 Forward nodes

**Comment by weslambert** [comment]
Have you first tried opening the firewall port on the manager using the Beats option via so-allow?

**Reply by john-mathew-itt** [reply]
No, I haven't tried that since I am able to receive data from heartbeat running on the forward node. I have set up ICMP heartbeat monitors that ping my servers at defined intervals of time. This data is output to manager logstash and the data shows up under **so-beats** index pattern. I haven't installed any heartbeat index template and the data does not show up in the Kibana Uptime app.

**Comment by john-mathew-itt** [comment, marked as answer]
I was able to finally achieve this by installing heartbeat on my forward node and loading the heartbeat elasticsearch template as a custom ES template as mentioned [here](https://github.com/Security-Onion-Solutions/securityonion/discussions/6004) on my managersearch node. After defining monitors in heartbeat, I was able to see the data in the Uptime app.

Regards,
John.
